common/bloodlines/wc_earned_bloodlines.txt

```diff
@@ -169,4 +169,67 @@ scarlet_king = { # Garithos Usurper Bloodline
     allow_bastards = no
     picture = GFX_bloodlines_placeholder_symbol
     flags = { created_bloodline bloodline_drill bloodline_flag_scarlet_king }		
+}
+
+# Beated the Scourge
+end_of_scourge = {
+	threat_decay_speed = 0.05
+
+	general_opinion = 3
+
+	levy_reinforce_rate = 0.1
+	army_reinforce_rate = 0.1
+
+	bloodline_effect_great_heroes_appear = 1
+
+	inheritance = patrilineal
+    allow_bastards = yes
+    picture = GFX_bloodlines_symbol_roman
+    flags = { created_bloodline end_of_scourge_bloodline_flag bloodline_heroes }
+	active = {
+		NOT = { 
+			religion = death_god
+		}
+	}
+}
+# Beated the Legion
+end_of_legion = {
+	threat_decay_speed = 0.05
+
+	general_opinion = 3
+
+	land_morale = 0.05
+	land_organisation = 0.1
+
+	bloodline_effect_great_heroes_appear = 1
+
+	inheritance = patrilineal
+    allow_bastards = yes
+    picture = GFX_bloodlines_symbol_roman
+    flags = { created_bloodline end_of_legion_bloodline_flag bloodline_heroes }
+	active = {
+		NOT = { 
+			religion = burning_legion_religion
+		}
+	}
+}
+# Beated the Black Empire
```

Comment:
Use an alternative for this, such as "Defeat of the Black Empire" beated holds no sense, nor is it even a proper word.

Reply:
Done in 7290a3007e3ad6d9014067a0cb55033534f6ccfe.